judge/views/problem.py

```diff
@@ -374,14 +374,14 @@ def apply_full_text(queryset, query):
         return queryset.search(query, queryset.BOOLEAN).extra(order_by=['-relevance'])
 
     def get_filter(self):
-        filter = Q(is_public=True) & Q(is_organization_private=False)
+        _filter = Q(is_public=True) & Q(is_organization_private=False)
```

Comment:
this is not related to the PR, please consider to not include non-related stuff to the PR